src/nvim/api/ui_events.in.h

```diff
@@ -167,4 +167,7 @@ void msg_history_show(Array entries)
   FUNC_API_SINCE(6) FUNC_API_REMOTE_ONLY;
 void msg_history_clear(void)
   FUNC_API_SINCE(10) FUNC_API_REMOTE_ONLY;
+
+void error_exit(Integer status)
```

Comment:
Any reason not to make this a more general "exit" event? The status signals whether it's an error or not.

Reply:
sure, could be done as a follow-up.